docs/graphs/graph-concepts/edge-vs-vertex.md

```diff
@@ -1,37 +1,26 @@
 ---
-title: Edge vs. Vertex
+title: Vertices and Edges
 sidebar_position: 40
 ---
 
-This page discusses what data should be in an edge and what should be in a vertex in the context of Macrometa graphs.
+This page discusses best practices for determining what data should be stored in edges and vertices in Macrometa graphs.
 
 ## Vertices
 
-The main objects in your data model, such as users, groups or articles, are usually considered to be vertices.
+Vertices are the main objects in your graph data model, such as users, groups, or articles. For each type of object, a [document collection](../../collections/documents/) (also called a vertex collection) stores the individual entities. Entities can be connected by edges to express and classify relations between vertices. It often makes sense to have one edge collection per relation type.
```

Comment:
We could provide some examples of edges: likes (as in user likes article), orders (as in user orders a product).